docs/docsite/rst/dev_guidelines.rst

```diff
@@ -481,19 +481,128 @@ should return in the module.  As well as information related to the call itself,
 some response metadata.  It is OK to return this to the user as well as they may find it useful.
 
 Boto3 returns all values CamelCased.  Ansible follows Python standards for variable names and uses
-snake_case. There is a helper function in module_utils/ec2.py called ``camel_dict_to_snake_dict``
+snake_case.  There is a helper function in module_utils/ec2.py called ``camel_dict_to_snake_dict``
```

Comment:
```suggestion
snake_case.  There is a commonly used helper function ``ansible.module_utils.common.dict_transformations.camel_dict_to_snake_dict``
```

This moved to the main Ansible code base back in Ansible 2.6.0, it's imported into the ec2 module for compatibility.  Probably time to clean up, but it's never been a priority...